src/QMCWaveFunctions/Fermion/SlaterDetBuilder.cpp

```diff
@@ -116,18 +116,12 @@ std::unique_ptr<WaveFunctionComponent> SlaterDetBuilder::buildComponent(xmlNodeP
       // to simplify the logic inside DiracDeterminantWithBackflow,
       // I'm requiring that only a single <backflow> block appears
       // in the xml file
-      if (BFTrans != 0)
-      {
-        APP_ABORT("Only a single backflow block is allowed in the xml. Please collect all transformations into a "
-                  "single block. \n");
-      }
-      UseBackflow = true;
-      // creating later due to problems with ParticleSets
-      //BFTrans = new BackflowTransformation(targetPtcl,ptclPool);
-      BFTrans = nullptr;
-      BFnode  = cur;
-      // read xml later, in case some ParticleSets are read from hdf5 file.
-      //BFTrans->put(cur);
+      if (BFTrans)
+        myComm->barrier_and_abort("Only a single backflow block is allowed in the xml. "
```

Comment:
Isn't this just a UniformCommunicateError? It appears myComm is only used to call this error mechanism I'd like to see far fewer objects dependent on Communicate.

Reply:
This is a sophisticated builder and should carry myComm. So call barrier_and_abort directory. I agree that classes which have myComm must be limited.